api/ca.proto

```diff
@@ -43,6 +43,10 @@ message IssueNodeCertificateRequest {
 	// Token represents a user-provided string that is necessary for new
 	// nodes to join the cluster
 	string token = 3;
+
+	// Spec is the spec of the node being requested, not all fields in
+	// spec will be honored.
```

Comment:
Let's include a list of which fields are honored. Right now it would just be "availability".